proposals/PROPOSALS.md

```diff
@@ -0,0 +1,156 @@
+
+If none of these applies, the proposal is Rejected.
+
+### Responsibilities of the members
+
+- Review all HIPs 
+- Play a role in the discussions, learn in advance about the topic if needed, and make up their mind in the voting process.
+
+### Guests
+
+Experts in some fields may be invited to specific meetings as guests when discussing related HIPs. Their input would be important to discuss the current state of the proposal, both its design and implementation.
+
+## Proposal states
+
+The state of a proposal changes over time depending on the phase of the process and the decisions taken by the HF Technical Track. A given proposal can be in one of several states:
+
+0. **Discussion:** The proposal is being discussed informally on discourse. In this phase the proposal only exists informally, no pull request exists yet.
+1. **Submitted:** A pull request has been opened against the proposals repo.
+2. **Validated:** The submitted proposal has been validated for conformity to the [proposal template](proposals/TEMPLATE.md), and made all necessary changes to the proposal name and its location in the repo.
+3. **Under review:** The proposal will be under review until the next available Tech Track meeting takes place.
+4. **Under revision:** Authors are addressing the issues pinpointed throughout the discussion and feedback process from the community and/or the HF Technical Track.
+5. **Dormant:** When a HIP has been under revision for more than two iterations (that is, no progress has been made since the last review), it’s considered dormant, in which case any related activity will be paralyzed and the HF Tech Track will not allocate more resources to it.
+6. **Postponed:** The HIP has been postponed under some concrete conditions. When these are met, the HIP can be resubmitted.
+7. **Rejected:** The HIP has been rejected with a clear and full explanation.
+8. **Merged:** The HIP has been accepted and has been merged into the repo.
+
+## How do I submit? ##
+
+The process to submit is simple:
+
+* Submit a Pre-HIP topic to the HF Discourse, soliciting informal feedback.
+* If the feedback is favorable, fork the Haskell Foundation tech-proposals repository, [https://github.com/haskellfoundation/tech-proposals](https://github.com/haskellfoundation/tech-proposals).
+* Create a new HIP file in the `hips/` directory. Use the [HIP template](https://github.com/haskellfoundation/tech-proposals/blob/main/TEMPLATE.md)
+ * Make sure the new file follows the format: `YYYY-MM-dd-{title}.md`. Use the current date for `YYYY-MM-dd`.
+ * Use the [Markdown Syntax](http://daringfireball.net/projects/markdown/syntax) to write your HIP.
+* Commit your changes to your forked repository
+* Create a new [pull request](https://github.com/haskellfoundation/tech-proposals/pull/new).
+* Notify the Haskell HIP team on the Slack instance or on Discourse (or both!).
```

Comment:
Is this last step necessary? I would expect GitHub to do this notification.

Reply:
I agree that GitHub should, but I also really like this step. It's a fallback in case someone erroneously forked a fork of this repo and sent the pull request there instead. It also establishes an extra line of communication to the author. You may not know their slack/discourse/email name otherwise. Good for comments like 'awesome proposal', or 'I got the impression that what you were discussing in Discourse was a different proposal?' or 'I don't think I understand any of this small-step semantics stuff, we may need to wait for goldfirere to get back from his trip so we can make sense of it'.